tests/qgis-projects/tests/attribute_table.qgs.cfg

```diff
@@ -64,26 +69,24 @@
             "maxScale": 1000000000000,
             "toggled": "False",
             "popup": "False",
-            "popupFrame": null,
             "popupSource": "auto",
             "popupTemplate": "",
             "popupMaxFeatures": 10,
             "popupDisplayChildren": "False",
             "popup_allow_download": true,
-            "noLegendImage": "False",
+            "legend_image_option": "hide_at_startup",
             "groupAsLayer": "False",
             "baseLayer": "False",
             "displayInLegend": "True",
             "group_visibility": [],
             "singleTile": "True",
             "imageFormat": "image/png",
             "cached": "False",
-            "serverFrame": null,
             "clientCacheExpiration": 300
         },
-        "quartiers": {
+        "Les quartiers \u00e0 Montpellier": {
             "id": "quartiers_5fe55662_2cbf_48f4_a505_498c61fe978c",
-            "name": "quartiers",
+            "name": "Les quartiers \u00e0 Montpellier",
```

Comment:
Is-it an encoding issue on the file ?
Should I do something ?

Reply:
I don't know. It is cleanly represented in the developer tool. I guess it is to avoid some issue w/ JSON format.
@rldhont an idea?

![image](https://github.com/3liz/lizmap-web-client/assets/2145040/513d1bbf-6e0b-4c38-a624-30a3ae2818b8)